modules/vault-cluster/main.tf

```diff
@@ -132,7 +132,7 @@ module "security_group_rules" {
 
   security_group_id                    = "${aws_security_group.lc_security_group.id}"
   allowed_inbound_cidr_blocks          = ["${var.allowed_inbound_cidr_blocks}"]
-  allowed_inbound_security_group_ids   = ["${var.allowed_inbound_security_group_ids}"]
+  allowed_inbound_security_group_ids   = "${var.allowed_inbound_security_group_ids}"
```

Comment:
Why remove the brackets here? I think this may lead to an error when multiple IDs are passed...

Reply:
Hrm. To be honest, I've gone back and forth on similar bits in my own code.  In variables.tf, allowed_inbound_security_group_ids is defined to be a list. When I removed the brackets my assumption was that, due to that definition, it should "just work". You've got me second-guessing myself but it *does* work in my usecase. I'll put the brackets back since it was clearly working & passing tests.